backstory/src/test/java/com/google/sps/images/AnnotatedImageTest.java

```diff
@@ -71,7 +71,7 @@ private static EntityAnnotation makeMockEntityAnnotationWithDescription(String d
    * @param fileType the file extension of the image.
    * @return the byte array representation of the image.
    */
-  private static byte[] getBytesFromImageReference(String reference, String fileType) throws IOException {
+  public static byte[] getBytesFromImageReference(String reference, String fileType) throws IOException {
```

Comment:
Changed this for re-usability (to not have to copy and paste this function for new tests).